README.md

```diff
@@ -90,7 +90,7 @@ Dependencies (libraries must be i686):
 - GCC
 - automake
 - autoconf
-
+- m4

```

Comment:
What is it used for?

Reply:
Needed for `autoconf`, but not always marked as a dependency. See 0f12f43 's message.